packages/react-components/src/templates/NetworkPageHeader.tsx

```diff
@@ -16,9 +18,27 @@ const containerStyles = css({
   }em `,
 });
 
+const textStyles = css({
+  maxWidth: `${610 / perRem}em`,
+});
+
 const controlsStyles = css({
-  display: 'flex',
-  justifyContent: 'flex-end',
+  display: 'grid',
+  gridRowGap: `${8 / perRem}em`,
+  gridColumnGap: `${18 / perRem}em`,
+  alignItems: 'center',
+  paddingTop: `${18 / perRem}em`,
+  [`@media (min-width: ${smallDesktopScreen.min}px)`]: {
+    paddingTop: `${2 / perRem}em`,
+    gridTemplateColumns: '230px auto',
```

Comment:
Can the toggle define this on its own based on its intrisic size, so that we only have to do `min-content` here?